test/spec/modules/sovrnBidAdapter_spec.js

```diff
@@ -1,291 +1,261 @@
-import {expect} from 'chai';
-import {spec} from 'modules/sovrnBidAdapter.js';
-import {config} from 'src/config.js';
```

Comment:
Could you please make all these format changes like excessive semicolons and apostrophes removal in a separate commit so we could focus on the actual logic change?

I would actually suggest a separate PR, but since this is a fork, a separate PR would be less conviniant